kn/fotos/entities.py

```diff
@@ -408,6 +409,12 @@ def update_metadata(self, parent, save=True):
 
         return True
 
+    def set_rotation(self, rotation, save=True):
+        self.rotation = rotation
+        self._data['caches'] = [] # invalidate cache
```

Comment:
Ja, dat zeg ik. Dat is de bedoeling. De cache is van de niet-geroteerde foto die dus verouderd is na rotatie.

De EXIF cache bestaat niet in die zin – bepaalde eigenschappen (afmetingen, rotatie en datum) zijn gewoon eigenschappen van de entity die in het geval van rotatie aangepast kunnen worden.